.travis.yml

```diff
@@ -104,6 +103,7 @@ jobs:
     # 64-bit OSX build with clang and run tests:
     - if: type != cron AND env(TRAVIS_EVENT_TYPE) != cron
       os: osx
+      osx_image: xcode11.3
```

Comment:
This is weird that this would be required here, while `dist: xenial` at top level applies to all jobs.

Reply:
Going by the example at https://docs.travis-ci.com/user/multi-os/#example-multi-os-build-matrix, both `dist` and `osx_image` should be part of the `jobs` section. I see that `xenial` is also the default linux dist (https://docs.travis-ci.com/user/reference/linux/#default); maybe we're getting our expected build env because of that. I can verify this in a separate PR.

Verified that we're indeed using xcode11.3 now: https://travis-ci.com/github/DynamoRIO/dynamorio/jobs/357194591.

Reply:
Not sure it's that clear: there are other examples which have a mix of global and jobs.include (and many examples with global plus jobs.exclude): https://docs.travis-ci.com/user/build-matrix/#explicitly-included-jobs-inherit-the-first-value-in-the-array

Mixing seems nicer: a global default with a few local overrides; avoid duplicating multiple settings in every single job.  I wonder if the xcode regression on Mar 2 was really global vs jobs.include or something else specific to osx.

`xenial` was not the default when the setting was put in place, of course.

Reply:
This also brings up other settings: `sudo`, `language`, etc.  The example I cited above has language settings both globally and in the matrix.  It seems odd that the os ones would have to be per-job but not the language.